resources/kubernetes/procore-abax/config.ts

```diff
@@ -7,4 +7,8 @@ export const tag = config.require('tag');
 export const image = config.require('image');
 
 export const sanityProjectId = config.requireSecret('sanity-project-id');
-export const sanityToken = config.requireSecret('sanity-token');
\ No newline at end of file
+export const sanityToken = config.requireSecret('sanity-token');
+export const procoreClientId = config.requireSecret('procore-app-client-id');
```

Comment:
```suggestion
export const procoreClientId = config.require('procore-app-client-id');
```

Should we make this visible, so that we know which one is live?

Reply:
I suppose that is a good approach